bukkit/src/main/java/net/skinsrestorer/bukkit/SkinsRestorer.java

```diff
@@ -338,16 +339,6 @@ public void pluginStartup() throws InitializeException {
         }
     }
 
-    private void initConfigAndStorage() throws InitializeException {
-        // Init config files
-        Config.load(dataFolderPath, getResource("config.yml"), srLogger);
-        // Set new default locale because we initialized localeManager early
-        localeManager.setDefaultLocale(LocaleParser.getDefaultLocale());
```

Comment:
These two lines were not added back above.

Reply:
I think the desired locale is already set by default, because we are passing it to `LocaleManager::create` after config is loaded
> `SkinsRestorerAPIShared.getApi().getDefaultForeign().getLocale()`

`DefaultForeignSubject::getLocale` uses the same `LocaleParser::getDefaultLocale`

```java
public class DefaultForeignSubject implements ISRForeign {
    @Override
    public Locale getLocale() {
        return LocaleParser.getDefaultLocale();
    }
}
```

Reply:
Well yes, but it won't work on players with a locale that wasn't translated for SkinsRestorer yet.
![image](https://user-images.githubusercontent.com/40795980/204005437-ee610990-bb73-45f6-8eab-086f29e68155.png)
getDefaultLocale() returns something else after config is loaded, so it should be updated on the locale manager.

Reply:
Actually, this will cause a lot of unintended behavior. May you just drop all your config loading changes and only focus on your key MySQL change?

Reply:
We are planning to split our configs in a future release to fix this config stuff.